its/projects/ConsoleMultiLanguage/ConsoleCSharp/ConsoleCSharp.csproj

```diff
@@ -5,7 +5,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{836183AA-5841-4A0E-ADC0-E337EEC543AB}</ProjectGuid>
-    <OutputType>Exe</OutputType>
+    <OutputType>Library</OutputType>
```

Comment:
We don't report for usage of `Console.WriteLine` on console applications so I changed the type to `Library`.